tools/cli/pkg/ers/model.go

```diff
@@ -26,7 +26,7 @@ type Instance struct {
 	Labels          string
 	LandscapeLabel  string
 	// TODO: Date comes in different format than specified
-	// ModifiedDate      date.Date //2021-10-04T112314.137Z
+	ModifiedDate   int //2021-10-04T112314.137Z
```

Comment:
```suggestion
	ModifiedDate   int64 //2021-10-04T112314.137Z
```
However I would suggest in general to use [time.Time](https://pkg.go.dev/time#Time) type that we have in the standard library for timestamps. However `2021-10-04T112314.137Z` looks malformed to me as to `RFC3339` it should be "2021-10-04T11:23:14.137Z"?

Example how to parse timestamp string:

```
package main

import (
	"fmt"
	"time"
)

type Instance struct {
	ModifiedDate time.Time
}

func main() {
	timeStampString := "2021-10-04T11:23:14.137Z"

	timeStamp, err := time.Parse(time.RFC3339, timeStampString)
	if err != nil{
		fmt.Println(err)
		return
	}

	instance := &Instance{}

	instance.ModifiedDate = timeStamp

	fmt.Println(instance.ModifiedDate)
} 
```